packaging/opensuse-42.2/snapd.changes

```diff
@@ -1,4 +1,9 @@
 -------------------------------------------------------------------
+Wed Aug 30 07:45:01 UTC 2017 - mvo@fastmail.fm
+
+- Update to upstream release 2.27.5
+
+-------------------------------------------------------------------
```

Comment:
was about to say "that doesn't look like a valid changelog entry" :-)

Reply:
That is a valid changelog entry for openSUSE